### ISSUE TITLE
What are the used compression plot sizes?

### ISSUE THREAD

**Issue by toto99303** [gabyvespasiano/chia-plot-calculator#1]
Hello,

I'm trying to use your calculator, but in reality I get different results.
Are you using gigahorse's compressed plot sizes?
I'm using bladebit cuda, for which the sizes are probably different.
For example bladebit C7 plot is around 78 GiB.
Can you have a look?

P.S. Thanks for the great calculator!

**Comment by gabyvespasiano** [comment]
Yes im use madmax , probably made one for bladebit but i dont have values
of bladebit

**Comment by toto99303** [comment]
Here's the sizes for bladebit:

<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>

Compression Level | GiB per plot | GB per plot
-- | -- | --
0 | 101.30 | 108.77
1 | 87.54 | 93.99
2 | 86.03 | 92.37
3 | 84.46 | 90.69
4 | 82.86 | 88.97
5 | 81.26 | 87.25
6 | 79.65 | 85.52
7 | 78.05 | 83.81
9 | 75.20 | 80.75


</body>
</html>

Source:
https://docs.google.com/spreadsheets/d/1k6c-OBDtggXqnEfOPdMmq3646puzvOD7dWojwCH2v3c
